src/orders/components/OrderProductsCardElements/OrderProductsTableRow.tsx

```diff
@@ -96,7 +96,7 @@ const TableLine: React.FC<TableLineProps> = ({
         {maybe(() => line.orderLine.productName) || <Skeleton />}
       </TableCellAvatar>
       <TableCell className={classes.colSku}>
-        {line?.orderLine.productSku || <Skeleton />}
+        {line?.orderLine ? line.orderLine.productSku : <Skeleton />}
```

Comment:
As I undestand `line.orderLine.productSku` can still be undefined, even though the `line?.orderLine` exists. What should be displayed then?

Reply:
Then nothing should be displayed, empty table cell.